test/integration_test/applicationbackup_test.go

```diff
@@ -32,21 +33,43 @@ func testApplicationBackup(t *testing.T) {
 	t.Run("postExecMissingRuleTest", applicationBackupRestorePostExecMissingRuleTest)
 	t.Run("preExecFailingRuleTest", applicationBackupRestorePreExecFailingRuleTest)
 	t.Run("postExecFailingRuleTest", applicationBackupRestorePostExecFailingRuleTest)
+	t.Run("labelSelector", applicationBackupLabelSelectorTest)
 	t.Run("scheduleTests", applicationBackupScheduleTests)
 }
 
 func triggerBackupRestoreTest(
 	t *testing.T,
 	appBackupKey []string,
+	additionalAppKeys []string,
 	appRestoreKey []string,
 	createBackupLocationFlag bool,
 	backupSuccessExpected bool,
+	backupAllAppsExpected bool,
 ) {
 	var err error
 	var ctxs []*scheduler.Context
+	var preRestoreCtx, postRestoreCtx *scheduler.Context
 	ctx := createApp(t, appKey)
 	ctxs = append(ctxs, ctx)
-	logrus.Infof("App created %v. Starting backup.", ctx.GetID())
+	var restoreCtx = &scheduler.Context{UID: ctx.UID, App: &spec.AppSpec{Key: ctx.App.Key, SpecList: []interface{}{}}}
```

Comment:
Split this over multiple lines so that it is easier to read
```
var restoreCtx = &scheduler.Context{
     UUID: ctx.UID,
     ...
}
```

Reply:
Fixed.

Reply:
Fixed.